lib/Suggestions.js

```diff
@@ -29,12 +50,12 @@ class Suggestions extends Component {
     const { props } = this;
     const suggestions = props.suggestions.map(function (item, i) {
       return (
-        <li key={i}
-        onMouseDown={props.handleClick.bind(null, i) }
-        onMouseOver={props.handleHover.bind(null, i) }
-        className={i == props.selectedIndex ? "active" : ""}>
-        <span dangerouslySetInnerHTML={this.markIt(item, props.query) } />
-        </li>
+          <li key={i}
+              onMouseDown={props.handleClick.bind(null, i) }
+              onMouseOver={props.handleHover.bind(null, i) }
+              className={i == props.selectedIndex ? "active" : ""}>
```

Comment:
If we were to use a string instead of an index for the string, this line could stand to be spread out over multiple lines for clarity. This is where the magic is happening anyway so good to make that very readable.